sdks/java/core/src/main/java/org/apache/beam/sdk/io/PubsubIO.java

```diff
@@ -648,7 +676,9 @@ private Bound(String name, PubsubSubscription subscription, PubsubTopic topic,
       @Override
       public void populateDisplayData(DisplayData.Builder builder) {
         super.populateDisplayData(builder);
-        populateCommonDisplayData(builder, timestampLabel, idLabel, topic);
+        PubsubTopic topicString =
+            topic == null ? null : topic.get();
```

Comment:
same point here as above, w.r.t. `topic.get`. Need also to check `topic.isAccessible()`.